### ISSUE TITLE
DEP0096: timers.unenroll() is deprecated. Please use clearTimeout instead.

### ISSUE THREAD

**Issue by muxator** [ether/etherpad-lite#3472]
Etherpad: d1481041c27b
Node: **10.4.0**

```
[2018-08-23 23:11:46.931] [ERROR] console - (node:30875) [DEP0096] DeprecationWarning: timers.unenroll() is deprecated. Please use clearTimeout instead.
```

**Comment by muxator** [comment]
This happens within ueberdb when using mysql, and is due to https://github.com/mysqljs/mysql/issues/2003.

That issue [was fixed in mysql 2.16.0](https://github.com/mysqljs/mysql/blob/master/Changes.md#v2160-2018-07-17), but ueberdb 0.4.0 still [pulls in 2.15.0](https://github.com/Pita/ueberDB/blob/f3369fdce3569c3f5b6718f1fae1f95d2f358428/package.json#L29).

**Comment by Wikinaut** [comment]
(same issue here)

**Comment by muxator** [comment]
This should have been fixed by 06d961f2b66d, which brought in an updated ueberDB version (**0.4.1**), which depends on updated mysql libraries (see https://github.com/ether/ueberDB/commit/9f0671b90ee9468099eee6b9047c46972e361a3f. It went from **2.15.0** to **2.18.1**).

Waiting for confirmation and then closing this.

**Comment by JohnMcLear** [comment]
Confirmed
